apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/CreateCopyJobOptionsTest.cs

```diff
@@ -27,13 +27,15 @@ public void ModifyRequest()
                 CreateDisposition = CreateDisposition.CreateIfNeeded,
                 WriteDisposition = WriteDisposition.WriteIfEmpty,
                 DestinationEncryptionConfiguration = new EncryptionConfiguration { KmsKeyName = "projects/1/locations/us/keyRings/1/cryptoKeys/1" },
-                OperationType = CopyOperationType.Clone
+                OperationType = CopyOperationType.Clone,
+                ConfigurationModifier = (options) => options.ETag = "test"
```

Comment:
Nit: there's no need for bracket round the lambda parameter, and it's non-idiomatic IMO. So just `options => options.ETag = "test"` would be better.

Reply:
(Same for all the other examples, I'm afraid...)

Reply:
done